man/org.freedesktop.login1.xml

```diff
@@ -525,8 +532,13 @@ node /org/freedesktop/login1 {
       using an RTC timer and hibernated. The only argument is the polkit interactivity boolean
       <varname>interactive</varname> (see below). The main purpose of these calls is that they enforce
       polkit policy and hence allow powering off/rebooting/suspending/hibernating even by unprivileged
-      users. They also enforce inhibition locks. UIs should expose these calls as the primary mechanism to
-      poweroff/reboot/suspend/hibernate the machine.</para>
+      users. They also enforce inhibition locks for non-root users. UIs should expose these calls
+      as the primary mechanism to poweroff/reboot/suspend/hibernate the machine. Additionally
+      <function>PowerOffWithInhibitors()</function>, <function>RebootWithInhibitors()</function>,
+      <function>HaltWithInhibitors()</function>, <function>SuspendWithInhibitors()</function>,
+      <function>HibernateWithInhibitors()</function>, <function>HybridSleepWithInhibitors()</function> and
+      <function>SuspendThenHibernateWithInhibitors()</function> are methods which support enforcing
```

Comment:
/which support enforcing inhibitors for root user/which will enforce respecting active inhibitors for privileged users too/